.circleci/config.yml

```diff
@@ -38,7 +38,7 @@ jobs:
           at: ~/
       - run:
           name: Lint
-          command: make lint-all
+          command: docker-compose up -d && docker-compose run --rm app bash -c "cd $(CKAN_HOME) && pip install --upgrade pip  && pip install flake8 && flake8 . --count --select=E9 --show-source --statistics"
```

Comment:
@anuveyatsu can you move this to a yarn command so that we can run this locally?

Reply:
@adborden locally, you can run this command via `yarn lint:python`. The reason we aren't using `yarn` in the CI job is that `yarn` isn't available in `lint_python_code` job as it uses "machine" as an executor. Read more about it here - https://circleci.com/docs/2.0/executor-types/#using-machine. There should be a way to get `yarn` there (eg, just install node and yarn probably) but I am not sure if it is something we need.

Reply:
We should make sure that CI is running the same commands that we run in development. Maybe `make` is a better choice than yarn for that reason.

Reply:
The machine executor already has a lot of tools built in. I think it already includes node.js so should just be `npm install -g yarn`.

Reply:
@adborden good point. I think there are other places where we could run "standard" commands. Should we work against existing issue or create a new one for it?